src/base.ts

```diff
@@ -48,6 +48,7 @@ export type ValidationError =
       nestedErrors?: unknown[];
     };
 
+// Not using simbols because of #813
```

Comment:
```suggestion
// Not using Symbol because of #813
```
Oops Italian grammar🤦🏻‍♂️